### PR TITLE
Add pokemon apearing on map

### PR DESCRIPTION
**Pull request by Reaver01**
Short Description: 

Fixes:
- 
- 
-

### CODE REVIEW

**Comment by douglascamata** [comment]
still has merge conflict issues

File: web/index.html [hunk 1]
```diff
@@ -17,237 +17,91 @@
...
+          console.log("Update Marker: Catchable Pokemon - " + parseFloat(cInfo['latitude']) + ", " + parseFloat(cInfo['longitude']));
+          cMarker.setPosition({lat: parseFloat(cInfo['latitude']), lng: parseFloat(cInfo['longitude'])});
+          cMarker.setIcon("image/icons/" + cInfo['pokemon_id'] + ".png");
+        }
+        if(cInfo['latitude'] == undefined && cMarker != undefined) {
+          cMarker.setIcon("image/icons/blank.png");
+        }
       }
       function updateInventory() {
         loadJSON('info.json',
```
Comment (solderzzc) [inline web/index.html]: You need this one to get rid of browser cache:
`
loadJSON('info.json?_=' + new Date().getTime(),
`

Comment (Reaver01) [inline web/index.html]: what line?

Comment (solderzzc) [inline web/index.html]: Line 103

**Pull request merged**